interpreter/k8s/src/main/scala/io/buoyant/transformer/k8s/DaemonSetTransformerInitializer.scala

```diff
@@ -31,11 +32,19 @@ case class DaemonSetTransformerConfig(
 
   private[this] val netmask = InetAddress.getByName("255.255.255.0")
 
+  private[this] val nodeLocal: (Address, Address) => Boolean = {
+    case (Address.Inet(_, a), Address.Inet(_, b)) => a.get("nodeName") == b.get("nodeName")
+    case _ => true
+  }
```

Comment:
seems slightly odd for this to be in the config class...

Should we have a MetadataGatewayTransformer?